es2015.js

```diff
@@ -24,6 +24,6 @@ module.exports = {
   CatchAllMessage: Message.CatchAllMessage,
 
   loadBot (adapterPath, adapterName, enableHttpd, botName, botAlias) {
-    return new Robot(adapterPath, adapterName, enableHttpd, botName, botAlias)
+    return new module.exports.Robot(adapterPath, adapterName, enableHttpd, botName, botAlias)
```

Comment:
it worked, but this way we can stub `Robot` in tests 
https://github.com/hubotio/hubot/pull/1359/files#diff-0740e6d4a0c565b774aca19f820a7550R193